tests/acceptance/30_generic_methods/file_check_FIFO_pipe.cf

```diff
@@ -0,0 +1,89 @@
+#########################################
+#
+# Test checking if a file is a FIFO or not 
+# 
+#########################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("$#this.promise_filename}") };
```

Comment:
Have opened https://www.rudder-project.org/redmine/issues/7907 for it, will change PR on the weekend.